bittensor/axon.py

```diff
@@ -88,7 +106,7 @@ def check_config(config: Munch) -> Munch:
                     config to check.
         """
         config = Nucleus.check_config(config)
-        logger.info('obtaining remote ip ...')
+        logger.info('optaining remote ip ...')
```

Comment:
typo. Why did we change it to optaining? lol

Reply:
lol